1-exercises/B-while-loop/exercise.js

```diff
@@ -6,9 +6,17 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+    let arr = [];
+    let i = 0;
+    while (i%2 === 0 && n > 0 && n > arr.length) {
+        arr.push(i);
+        i+=2;
+    }
+    return arr.toString();
 }
 
```

Comment:
hey, Saf in line 11 when you give the condition while loop you can make it simply (n > arr.length) and it will work!